integration/integration_test.go

```diff
@@ -103,7 +103,7 @@ func TestIntegration(t *testing.T) {
 				"workspace.transition":              `start`,
 				"workspace_owner.email":             `testing@coder\.com`,
 				"workspace_owner.full_name":         `default`,
-				"workspace_owner.groups":            `\[\]`,
+				"workspace_owner.groups":            `\[\"Everyone\"\]`,
```

Comment:
Integration test doesn't appear to like this. Should this be moved to a separate PR?

Reply:
That's annoying, I thought this would appease.. I'll make a new PR.